ci-operator/jobs/shiftstack/bugwatcher/shiftstack-bugwatcher-main-presubmits.yaml

```diff
@@ -0,0 +1,98 @@
+presubmits:
+  shiftstack/bugwatcher:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - main
+    cluster: build01
+    context: ci/prow/images
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-shiftstack-bugwatcher-main-images
+    rerun_command: /test images
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --target=[images]
```

Comment:
Does this rebuild the bugwatcher image? Does it persist it, or simply check that the image can be built? If the former, wouldn't we want this as a 'post' job?

Reply:
As a general rule, you probably want fresh dependencies in a periodic, even when you don't merge stuff often